### PR TITLE
[B2BP-288] Fixed minor issues with Footer and Feature components

### PR DESCRIPTION
**Pull request by simosalsi**
Above mentioned issues:
- Footer on Strapi's side had extra MuiSocialIcons not accepted by NextJS
- Feature component would not show icons correctly

#### List of Changes

- Removed extra MuiSocialIcons from Strapi
- Converted Feature component to use a standard string instead of the `Icon` component

#### Motivation and Context

Bug fixes.

#### How Has This Been Tested?

Tested locally.

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .changeset/sweet-elephants-joke.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"nextjs-website": patch
+"strapi-cms": patch
+---
+
+fix icons from Features component on Next.js plus fix footer socials link on strapi
```
Comment (datalek) [inline .changeset/sweet-elephants-joke.md:6]: Rather than having a description that includes all the changes in different packages, it is better to have a dedicated description per package. So it is best to create a changeset for each package. Just put them in different changeset files

File: apps/nextjs-website/src/components/Feature.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 'use client';
 import { Feature as FeatureEC } from '@pagopa/pagopa-editorial-components';
 import { FeatureProps } from '@pagopa/pagopa-editorial-components/dist/components/Feature/Feature';
-import { Icon } from '@mui/material';
-import { formatValidMuiIcon, isValidMuiIcon } from './Icons';
+// Only temporarily importing every icon. A task is planned to sub this for a restricted set of accepted icon names.
+import * as MuiIcons from '@mui/icons-material';
+import { isValidMuiIcon } from './Icons';
```
Comment (datalek) [inline apps/nextjs-website/src/components/Feature.tsx:6]: These imports shouldn't be required if you limit the possible values ​​for [icon](https://github.com/pagopa/b2b-portals/blob/d2d31b8b31a65afe3d42507b6c80384cee5519ae/apps/nextjs-website/src/lib/fetch/types/PageSection.ts#L72) as follows:

```typescript
      icon: t.union([
        t.literal('Instagram'),
        t.literal('LinkedIn'),
        t.literal('Facebook'),
        t.literal('Twitter'),
        t.null,
      ]),
```

File: apps/nextjs-website/src/components/Feature.tsx [hunk 3]
```diff
@@ -13,7 +14,7 @@ const makeFeatureProps = ({
   items: items.map((item) => ({
     stackIcon: {
       ...(isValidMuiIcon(item.icon) && {
-        startIcon: <Icon>{formatValidMuiIcon(item.icon)}</Icon>,
+        icon: item.icon as keyof typeof MuiIcons,
```
Comment (datalek) [inline apps/nextjs-website/src/components/Feature.tsx:17]: limiting the values for icon property allows to remove the cast

**Review by datalek — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/components/Feature.tsx (hunk 2 above)
Comment (simosalsi) [reply apps/nextjs-website/src/components/Feature.tsx:6]: The icons allowed in the Feature section are not reduced to just social icons, that's only for the Footer's socialLinks.

We will reduce the icons allowed for the Feature Section Items with task [B2BP-348](https://pagopa.atlassian.net/jira/software/projects/B2BP/boards/482?selectedIssue=B2BP-348)

File: apps/nextjs-website/src/components/Feature.tsx (hunk 3 above)
Comment (datalek) [inline apps/nextjs-website/src/components/Feature.tsx:17]: You don't need the cast if `isValidMuiIcon` returns a [type predicate](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates). You can just change [isValidMuiIcon](https://github.com/pagopa/b2b-portals/blob/93e0be0/apps/nextjs-website/src/components/Icons/index.ts#L4-L5) as follows:

```typescript
export const isValidMuiIcon = (iconName?: string | null): iconName is (keyof typeof MuiIcons) =>
  !!iconName && iconName in MuiIcons;
```

**Review by datalek — APPROVED** [review]
💥
